packages/atomic/src/components/facets-v1/atomic-color-facet/atomic-color-facet.tsx

```diff
@@ -20,8 +20,8 @@ import {FacetPlaceholder} from '../../facets/atomic-facet-placeholder/atomic-fac
 import {FacetContainer} from '../facet-container/facet-container';
 import {FacetHeader} from '../facet-header/facet-header';
 import {FacetSearchInput} from '../facet-search/facet-search-input';
-import {FacetValueCheckbox} from '../facet-value-checkbox/facet-value-checkbox';
-import {FacetValueBox} from '../facet-value-box/facet-value-box';
+import {FacetValueCheckboxColor} from '../facet-value-checkbox-color/facet-value-checkbox-color';
+import {FacetValueBoxColor} from '../facet-value-box-color/facet-value-box-color';
```

Comment:
Would placing `Color` at the start of the name read more naturally?

`ColorFacetValueCheckbox` or even shortening to `ColorFacetCheckbox`

Reply:
It reads more naturally and the actual facet is called `Color Facet` so it will match accordingly